.github/actions/environment/action.yml

```diff
@@ -85,4 +85,30 @@ runs:
         # files from previous runs, to avoid of it need to run:
         git clean -xffd
         git submodule foreach --recursive 'git clean -xffd'
+
+        # A temporary workaround that should be removed after the corresponding
+        # issue is fixed. See for details:
+        #   https://github.com/tarantool/tarantool/issues/7472.
+        if [ "${{ github.workflow }}" = "coverage" ]; then
+            LUAJIT_TEST_VARDIR=/tmp/luajit-test-vardir
```

Comment:
Minor: Is it possible to use `{{ runner.temp }}` here? Feel free to ignore.

Reply:
I realised that this dir can be used by other workflows (we have shared runners across a few repos) and it is not good to mount this dir to 1GB partition, someone may fail on this. Let's use dirs created by us.